javascript-conditionals/main.js

```diff
@@ -0,0 +1,134 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+function isUnderFive(number) {
+  if (number >= 5) {
+    return false;
+  } else if (number < 5) {
+    return true;
+  }
+}
+console.log('isUnderFive Output ==>', isUnderFive(4));
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+console.log('isEven Output ==> ', isEven(10));
+
+function startsWithJ(string) {
+  const str = string.toUpperCase();
```

Comment:
Remove this line of code. We don't want to convert the whole string to uppercase, we want to check if the current string has a capital J. When you convert the whole string to uppercase, if the function received a value of 'javaScript', this would return `true`, when it should return `false`.

We want to check if the **original** string starts with a capital 'J'